mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/veilederflate/VeilederflateTiltakQueries.kt

```diff
@@ -105,8 +112,8 @@ private fun Row.toVeilederflateTiltaksgjennomforing(): VeilederflateTiltakGruppe
             selskapsnavn = stringOrNull("arrangor_navn"),
             kontaktpersoner = arrangorKontaktpersoner,
         ),
-        fylke = string("nav_region"),
-        enheter = navEnheter,
+        fylker = fylker,
```

Comment:
Ser ut som at dette er endring i en modell som også er eksponert i `openapi.yaml`. Er selve feltet brukt i i klienten eller i veilederflate? Burde sørge for at vi ikke brekker noe der heller.